### PR TITLE
Pattyssilveira/tooltip component

### PR DESCRIPTION
**Pull request by pattyssilveira**
this PR adds:
-Tooltip component and storybook
-React-Popover library

### CODE REVIEW

File: components/atoms/tooltip/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Popover from 'react-popover'
+
+import styles from './styles.css'
+
+class Tooltip extends React.Component {
```
Comment (beatriz1304) [inline components/atoms/tooltip/index.js]: Import Component

Comment (beatriz1304) [reply components/atoms/tooltip/index.js]: It's not wrong but we're importing `Component` so I think it's important to keep the consistency

File: components/atoms/tooltip/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    isOpen: true,
+  }
+
+  toggle(toState = null) {
+    this.setState({ isOpen: toState === null ? !this.state.isOpen : toState })
+  }
+  render() {
+    const { isOpen } = this.state
+    return (
+      <Popover className={styles.tooltiptext} isOpen={isOpen} body="o ousado chegou kkkkjjk">
```
Comment (beatriz1304) [inline components/atoms/tooltip/index.js]: Remove the hardcoded message on `body` prop.
You need to receive this as a prop

File: components/atoms/tooltip/index.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import Popover from 'react-popover'
+
+import styles from './styles.css'
+
+class Tooltip extends React.Component {
+  state = {
+    isOpen: true,
+  }
+
+  toggle(toState = null) {
```
Comment (beatriz1304) [inline components/atoms/tooltip/index.js]: I don't know if it's necessary to assign `toState`variable with null

Comment (beatriz1304) [reply components/atoms/tooltip/index.js]: I think you can rename this parameter to a clear name

File: components/atoms/tooltip/index.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          onMouseOut={() => this.toggle(false)}
+        >
+          {this.props.children}
+        </div>
+      </Popover>
+    )
+  }
+}
+
+Tooltip.propTypes = {
```
Comment (beatriz1304) [inline components/atoms/tooltip/index.js]: You should define all your properties here

Comment (beatriz1304) [reply components/atoms/tooltip/index.js]: and when the entity is a class we define the `propTypes` and `defaultProps`as a static object, like below to keep the consistency: 

`static propTypes = {
    navFooterIcon: PropTypes.string,
    children: PropTypes.node.isRequired,
    ....
  }`

File: components/atoms/tooltip/styles.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
+.tooltip {
+    position: relative;
+    width: max-content;
+    padding: 10px;
+}
+
+ .tooltiptext {
```
Comment (beatriz1304) [inline components/atoms/tooltip/styles.css]: Use` camel case` for `CSS variables`

File: components/atoms/tooltip/styles.css [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    background-color: black;
+    color: var(--color-white);
+    text-align: center;
+    padding: 5px 10px;
+    border-radius: 6px;
+}
+
+:global .Popover-tip {
```
Comment (beatriz1304) [inline components/atoms/tooltip/styles.css:16]: Use `camel case` for `CSS variables `

File: components/atoms/tooltip/index.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+import styles from './styles.css'
+
+class Tooltip extends React.Component {
+  state = {
+    isOpen: true,
+  }
+
+  toggle(toState = null) {
+    this.setState({ isOpen: toState === null ? !this.state.isOpen : toState })
```
Comment (beatriz1304) [inline components/atoms/tooltip/index.js]: I think you can use `prevState` like this code below

``sideMenuHandler = () => {
    this.setState(prevState => ({
      sidenavIsOpen: !prevState.sidenavIsOpen,
    }))
  }
``

File: components/atoms/tooltip/styles.css [hunk 8]
```diff
@@ -0,0 +1,21 @@
+.tooltip {
+    position: relative;
+    width: max-content;
+    padding: 10px;
+}
+
+ .tooltiptext {
+    background-color: black;
```
Comment (beatriz1304) [inline components/atoms/tooltip/styles.css]: Change `black` to a variable

**Comment by pattyssilveira** [comment]
@beatriz1304 I made some modifications to the code

**Review by dleitee — DISMISSED** [review]

**Pull request merged**